drivers/mmcsd/sdio.c

```diff
@@ -460,15 +460,15 @@ int sdio_enable_function(FAR struct sdio_dev_s *dev, uint8_t function)
       return ret;
     }
 
-  /* Wait 10ms for function to be enabled */
+  /* Wait 1s for function to be enabled */
 
-  int loops = 10;
+  int loops = 100;
 
   while (loops-- > 0)
     {
-      up_mdelay(1);
+      up_mdelay(10);
```

Comment:
can we use `nxsig_usleep()` here? Or the scheduler is not running at this moment?

Reply:
emm....You are right, thanks for the comment, `nxsig_usleep()` is better

Reply:
Done